addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2611,6 +2599,18 @@ const Many2oneUserValueWidget = SelectUserValueWidget.extend({
             method: 'read',
             args: [recTuples.map(([id, _name]) => id), this.options.fields],
         });
+        // Remove select options.
+        this._userValueWidgets.filter(widget => {
```

Comment:
Not able to reproduce the exact issue you described but this is obviously better and I am sure what you describe happens anyway :wink: